src/main/resources/SLING-INF/content/products.json

```diff
@@ -0,0 +1,4 @@
+{
+  "jcr:primaryType": "sling:Folder",
+  "sling:resourceType": "pantheon/product/all"
```

Comment:
I wouldn't bother with the resource type here. I don't think we're going to use it for these nodes.

Reply:
line 3 is removed.